### DISCUSSION TITLE
Sound detection esp32 or raspberry pi

### DISCUSSION THREAD

**Discussion by ignacio82** [blakeblackshear/frigate#10465] [Ideas]
I would love to be able to use an [esp32 atom echo](https://shop.m5stack.com/products/atom-echo-smart-speaker-dev-kit) to stream audio to my frigate and have use frigate to detect when my dog is barking, baby crying, etc. In many cases I would actually prefer that to a camera. Is this possible? If so, is it in the roadmap? Alternatively, I would be happy to do the same thing with a raspberry pi if that is easier.

**Comment by NickM-27** [comment]
this is already possible, frigate can work with any stream source for audio.

**Reply by ignacio82** [reply]
Thanks @NickM-27. Could you point me to a youtube video, or blogpost that goes over who to do this?
